cache/cache_test.cc

```diff
@@ -933,15 +933,15 @@ TEST_P(CacheTest, DefaultShardBits) {
   // test1: set the flag to false. Insert more keys than capacity. See if they
```

Comment:
Changes to lru_cache.h/cc, fast_lru_cache.h/cc and clock_cache.h/cc are syntactic replacements of ShardedCache (resp. CacheShard) for ShardedCache32 (resp. CacheShard32).